src/rulesets/LinearAlgebra/symmetric.jl

```diff
@@ -114,7 +119,14 @@ function frule(
     ∂K = mul!(ΔA.data, tmp, U)
     ∂Kdiag = @view ∂K[diagind(∂K)]
     ∂λ = real.(∂Kdiag)
-    ∂K ./= λ' .- λ
+    # in-place ∂X = sylvester(Λ, -Λ, ∂K),
+    broadcast!(∂K, ∂K, transpose(λ), λ) do ∂Kij, λj, λi
+        Δλ = λj - λi
+        # when Δλ=0, then ∂Xij * Δλ = ∂Kij only has a solution when ∂Kij=0.
+        # of infinite solutions, choose ∂Xij = 0
+        iszero(Δλ) && iszero(∂Kij) && return zero(∂Kij)
```

Comment:
isn't
```suggestion
        iszero(∂Kij) && return ∂Kij
```
surficient?